packages/cli/generators/app/templates/index.js.ejs

```diff
@@ -8,6 +8,9 @@ if (require.main === module) {
     rest: {
       port: +(process.env.PORT || 3000),
       host: process.env.HOST,
+      // Provide a graceful close for http/https servers
+      // See https://www.npmjs.com/package/stoppable
```

Comment:
Please include the information from code comment:

> Defaults to Infinity (don't force-close). If you want to immediately kill all sockets you can use a grace of 0.

Alternatively, add a link to our docs where this configuration option is described (_`rest` options_ section in `Server.md`).